### PR TITLE
Normal generator

### PR DESCRIPTION
**Pull request by mchernov-intel**

### CODE REVIEW

File: cpp/oneapi/dal/test/engine/dataframe_common.hpp [hunk 1]
```diff
@@ -259,6 +259,7 @@ class dataframe_builder {
             : impl_(new dataframe_builder_impl{ row_count, column_count }) {}
 
     dataframe_builder& fill_uniform(double a, double b, std::int64_t seed = 7777);
+    dataframe_builder& fill_normal(double a, double b, std::int64_t seed = 7777);
```
Comment (rlnx) [inline cpp/oneapi/dal/test/engine/dataframe_common.hpp:262]: Use more appropriate var names for normal distribution, e.g., center/loc and std

File: cpp/oneapi/dal/test/engine/dataframe_common.cpp [hunk 2]
```diff
@@ -261,6 +261,47 @@ class dataframe_builder_action_fill_uniform : public dataframe_builder_action {
     std::int64_t seed_ = 7777;
 };
 
+class dataframe_builder_action_fill_normal : public dataframe_builder_action {
```
Comment (rlnx) [inline cpp/oneapi/dal/test/engine/dataframe_common.cpp:264]: Merge with master and move this class to dataframe_actions.hpp

**Comment by mchernov-intel** [comment]
moved to #1449

**Pull request closed without merging**